python/ray/serve/tests/BUILD

```diff
@@ -65,7 +65,6 @@ py_test_module_list(
     "test_regression.py",
     "test_request_timeout.py",
     "test_cluster.py",
-    "test_autoscaling_policy.py",
```

Comment:
let's leave the version w/o feature flag set here and in the added one-off test below set the feature flag environment variable

Reply:
suggesting relevant change below